tools/truncated/package.json

```diff
@@ -0,0 +1,76 @@
+{
+    "name": "@spectrum-web-components/truncated",
+    "version": "0.41.1",
+    "publishConfig": {
+        "access": "public"
+    },
+    "description": "Web component implementation of a Spectrum design Truncated",
+    "license": "Apache-2.0",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/adobe/spectrum-web-components.git",
+        "directory": "packages/truncated"
+    },
+    "author": "",
+    "homepage": "https://adobe.github.io/spectrum-web-components/components/truncated",
+    "bugs": {
+        "url": "https://github.com/adobe/spectrum-web-components/issues"
+    },
+    "main": "src/index.js",
+    "module": "src/index.js",
+    "type": "module",
+    "exports": {
+        ".": {
+            "development": "./src/index.dev.js",
+            "default": "./src/index.js"
+        },
+        "./package.json": "./package.json",
+        "./src/Truncated.js": {
+            "development": "./src/Truncated.dev.js",
+            "default": "./src/Truncated.js"
+        },
+        "./src/index.js": {
+            "development": "./src/index.dev.js",
+            "default": "./src/index.js"
+        },
+        "./src/truncated.css.js": "./src/truncated.css.js",
+        "./sp-truncated.js": {
+            "development": "./sp-truncated.dev.js",
+            "default": "./sp-truncated.js"
+        }
+    },
+    "scripts": {
+        "test": "echo \"Error: run tests from mono-repo root.\" && exit 1"
+    },
+    "files": [
+        "**/*.d.ts",
+        "**/*.js",
+        "**/*.js.map",
+        "custom-elements.json",
+        "!stories/",
+        "!test/"
+    ],
+    "keywords": [
+        "spectrum css",
+        "web components",
+        "lit-element",
+        "lit-html"
+    ],
+    "dependencies": {
+        "@spectrum-web-components/base": "^0.41.0",
+        "@spectrum-web-components/overlay": "^0.41.0",
+        "@spectrum-web-components/styles": "^0.41.0",
+        "@spectrum-web-components/tooltip": "^0.41.0"
+    },
+    "devDependencies": {
+        "@open-wc/testing": "^3.2.0",
+        "@web/test-runner-commands": "^0.9.0"
+    },
+    "types": "./src/index.d.ts",
+    "customElements": "custom-elements.json",
+    "sideEffects": [
+        "./sp-*.js",
+        "./**/*.dev.js",
```

Comment:
This is wrong like this somewhere in our tooling 🙈. Please you remove the duplicate.